cache_helper/decorators.py

```diff
@@ -18,7 +20,12 @@ def wrapper(*args, **kwargs):
 
             if value is None:
                 value = func(*args, **kwargs)
-                cache.set(key, value, timeout)
+                # Try and set the key, value pair in the cache.
+                # But if it fails on a Memcached Error handle it.
+                try:
+                    cache.set(key, value, timeout)
+                except MemcachedError:
```

Comment:
afraid this won't fly as it is dependent on one of two memcached bindings (and django cache can work with db caching as well).  

Is there more a package agnostic error that would cover your needs here?